internal/expr/parser.go

```diff
@@ -400,36 +421,39 @@ func (p *Parser) skipBlanks() bool {
 func (p *Parser) skipString(s string) bool {
 	if p.pos+len(s) <= len(p.input) &&
 		strings.EqualFold(p.input[p.pos:p.pos+len(s)], s) {
+		// Manually advance the parser to the end of the string.
 		p.pos += len(s)
+		var size int
+		p.char, size = utf8.DecodeRuneInString(p.input[p.pos:])
+		p.nextPos = p.pos + size
 		return true
 	}
 	return false
 }
 
-// isNameByte returns true if the given byte can be part of a name. It returns
+// isNameChar returns true if the given char can be part of a name. It returns
 // false otherwise.
-func isNameByte(c byte) bool {
-	return 'A' <= c && c <= 'Z' || 'a' <= c && c <= 'z' ||
-		'0' <= c && c <= '9' || c == '_'
+func isNameChar(c rune) bool {
+	return unicode.IsLetter(c) || unicode.IsDigit(c) || c == '_'
 }
 
-// isNameInitialByte returns true if the given byte can appear at the start of a
+// isNameInitialChar returns true if the given char can appear at the start of a
 // name. It returns false otherwise.
-func isInitialNameByte(c byte) bool {
-	return 'A' <= c && c <= 'Z' || 'a' <= c && c <= 'z' || c == '_'
+func isInitialNameChar(c rune) bool {
```

Comment:
The function name and doc-comment name are slightly out of alignment

Reply:
Nice spot.